js/views/userPageVw.js

```diff
@@ -1413,6 +1437,92 @@ module.exports = baseVw.extend({
 
   },
 
+  addModeratorClick: function(e){
+    var $targ = $(e.target).closest('.js-addmoderator');
+
+    $targ.addClass('loading');
+
+    var user = this.model.get('user');
+    var self = this;
+
+    var modList = {};
+    modList.moderators = user.moderators;
+    modList.moderators[modList.moderators.length] = this.userProfile.get('profile').guid;
+    user.moderators = modList.moderators;
+
+    saveToAPI('', user, user.serverUrl + "settings",
+      function(){
+        // confirmed
```

Comment:
This is the success function, you should update the status of the page here. If the call fails, you don't need to update anything. 

You can also simplify it, something like this:

function(){
        self.options.userModel.fetch({
          success: function(model, response) {
            if (self.isRemoved()) return;
            self.model.set('user', model.toJSON());
            self.getIsModerator();
          }
        });
      }

You don't need $targ.removeClass('loading'); in the success function, since it's in the always function, which is always called.